packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -54,15 +60,22 @@ export default class EditorEnvelope extends PureComponent {
         if (editorDefinition && editorDefinition.component) {
             const EditorComponent = editorDefinition && editorDefinition.component;
 
+            const restProps = omit(this.props, ['validationErrors', 'highlight']);
+            const {highlight, validationErrors} = this.props;
+            const isInvalid = validationErrors && validationErrors.length > 1;
+
+            // We pass down a classname to render a highlight status on the editor field
+            const classNames = mergeClassNames({
+                [style['envelope--highlight']]: highlight && !isInvalid,
+                [style['envelope--invalid']]: validationErrors && validationErrors.length > 0
```

Comment:
Shouldn't this be the the same as `isInvalid` ?

Reply:
fixed